src/archiefvernietigingscomponent/templates/start_page/index.html

```diff
@@ -0,0 +1,120 @@
+{% extends "start_page/master.html" %}{% load demo_tutorial %}
```

Comment:
I don't quite understand why to have two separate `master.html` tempates. Couldn't `start_page.html` be based on the same template as other pages?

Reply:
I merged it now!